lib/cryptoexchange/exchanges/acx/services/market.rb

```diff
@@ -0,0 +1,45 @@
+module Cryptoexchange::Exchanges
+  module Acx
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all output
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Acx::Market::API_URL}/tickers"
+        end
+
+        def adapt_all(output)
+          output.map do |market_pair_key, ticker|
+            adapt(ticker)
+          end
+        end
+
+        def adapt(output)
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = output['base_unit']
```

Comment:
I would suggest just passing in `market_pair` and call `base` and `target` for assignment?
This will ensure the base/target are all uppercased

Reply:
Sure, done.